src/layouts/stressPatientDashboard/stressPatientDashboard.js

```diff
@@ -0,0 +1,18 @@
+import './stressPatientDashboard.css'
+import Header from '../../components/header/header.js'
+import Title from '../../components/title/titleGraphPatient.js'
+import StessComment from '../../components/stressComment/stressComment.js'
+import Graph from '../../components/graph/graph'
+
+function StressPatientDashboard() {
+  return (
+    <div className="body">
+      <Header />
+      <Title />
+      <Graph />
+      <StessComment />
```

Comment:
It would be really cool if we could refactor this to some kind of `BaseLayout` component. This could contain the `Header`, `Title` and `props` (which would equal the `Graph` and `StressComment` in this case. In the future, it might also hold a footer. This `BaseLayout` component might look something like this. 


```js
function BaseLayout = (props) => {
   return (
    <div>
        <Header />
        <Title />
        {props.children}             
    </div>
  )
}
```

Reply:
It would be but mainly working on implementing and refactoring in a future sprint where we can add this.